src/components/Header/TopMainNav/SearchBar/SuggestionBox.tsx

```diff
@@ -38,25 +39,35 @@ export const SuggestionBox = ({ results, isActive }: Props) => {
 
   return (
     <div aria-label="suggestions" className={container}>
-      <div className="font-light text-dark-grey uppercase text-menu3 px-16">Results from docs</div>
+      <div className="font-light text-dark-grey uppercase text-menu3 px-16 mb-2">Results from docs</div>
       {results &&
         results.map((hit, index) => {
           const { title, highlight, url, id } = hit;
+          const [pageTitle, ...breadcrumbs] = title.split(' / ').filter(
+            (item) =>
+              // We need to get rid of 'Docs' because it's redundant and
+              // 'Root` because there are no root section it gets redirected to '/docs/'
+              !DOCUMENTATION_PATH.toLowerCase().includes(item.trim().toLowerCase()) ||
```

Comment:
item.trim().toLowerCase could be a const here maybe?

Reply:
This could also be `!DOCUMENTATION_NAME === item.trim().toLowerCase()` I think

Reply:
I tried, but it needs to be `.includes`, because `DOCUMENTATION_PATH` has `/`s at the end and the start of the string. So I'm pretty much using the string as an array.

Reply:
Ah yeah, that's why I said DOCUMENTATION_NAME - it's actually a different constant but doesn't really matter